### PR TITLE
Renaming Folder for Ingestion Client

### PR DESCRIPTION
**Pull request by PanosPeriorellis**
- renamed samples/ingestion from samples/batch
- renamed batch-ingestion-client to simply ingestion-client
-updated all docs and references

## Purpose

* ...

## Pull Request Type
What kind of change does this Pull Request introduce?

```
[ ] Bugfix
[ ] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[x ] Documentation content changes
[x] Other... Please describe:
```
file structure and folder names also changed

### CODE REVIEW

File: samples/ingestion/README.md [hunk 1]
```diff
@@ -0,0 +1,14 @@
+# Examples to use the Speech Services Transcription at scale
+
+Azure Speechis designed to handle a large number of audio fragments in storage, such as Azure Blobs. The functionality is exposed through a REST API and is easy to access from many programming languages. Some samples here do **NOT** require the installation of the Cognitive Service Speech SDK, but use the REST API directly or through additional helper libraries.
```
Comment (HenryvanderVegte) [inline samples/ingestion/README.md:3]: typo

File: samples/ingestion/ingestion-client/Setup/guide.md [hunk 2]
```diff
@@ -1,42 +1,42 @@
-# Getting started with the Batch Ingestion Client
+# Getting started with the Ingestion Client
 
-Get your audio files automatically transcribed without writing any code what-so-ever! The Batch Ingestion Client will monitor your dedicated Azure Storage container so that new audio files are transcribed automatically as soon as they land.
+Get your audio files automatically transcribed without writing any development effort! The Ingestion Client will monitor your dedicated Azure Storage container so that new audio files are transcribed automatically as soon as they land.
```
Comment (HenryvanderVegte) [inline samples/ingestion/ingestion-client/Setup/guide.md:3]: ... without any development effort

File: samples/ingestion/ingestion-client/Setup/guide.md [hunk 3]
```diff
@@ -1,42 +1,42 @@
...
 
 # Architecture
 
-This is a smart client in the sense that it implements best practices and optimized against the capabilities of the Azure Speech infrastructure. It utilizes Azure resources such as Service Bus and Azure Functions to orchestrate transcription requests to Azure Speech Services from audio files landing in your dedicated storage containers. Do you need more than transcription? Do you need to applly Sentiment to your transcript? Downstream analytics are possible too, with Text Analytics Sentiment and Redaction being offered as part of this solution too. 
+This is a smart client in the sense that it implements best practices and optimized against the capabilities of the Azure Speech infrastructure. It utilizes Azure resources to orchestrate transcription requests to Azure Speech Services from audio files landing in your dedicated storage containers. Do you need more than transcription? Do you need to applly Sentiment to your transcript? Downstream analytics are possible too, with Text Analytics Sentiment and Redaction being offered as part of this solution too.
```
Comment (HenryvanderVegte) [inline samples/ingestion/ingestion-client/Setup/guide.md:11]: ... and is optimized against...

Comment (HenryvanderVegte) [inline samples/ingestion/ingestion-client/Setup/guide.md:11]: apply

Comment (HenryvanderVegte) [inline samples/ingestion/ingestion-client/Setup/guide.md:11]: 2x "too" in the sentence, looks a bit weird imo

File: samples/ingestion/ingestion-client/Setup/guide.md [hunk 4]
```diff
@@ -1,42 +1,42 @@
...
+The diagram is simple and hopefully self-explanatory. As soon as files land in a storage container, the Grid Event that indicates the complete upload of a file is filtered and pushed to a Service bus topic. Azure Functions (time triggered by default) pick up those events and act, namely creating Tx requests using the Azure Speech Services batch pipeline. When the Tx request is successfully carried out, an event is placed in another queue in the same service bus resource. A different Azure Function triggered by the completion event starts monitoring transcription completion status and copies the actual transcripts in the containers from which the audio file was obtained. This is it. The rest of the features are applied on demand. Users can choose to apply analytics on the transcript, produce reports or redact, all of which are the result of additional resources being deployed through the ARM template. The solution will start transcribing audio files automatically and at scale. If -however- you want to customize further the Ingestion Client is possible too. The code is available in this repo.
```
Comment (HenryvanderVegte) [inline samples/ingestion/ingestion-client/Setup/guide.md:17]: I'd just remove the first sentence

Comment (HenryvanderVegte) [inline samples/ingestion/ingestion-client/Setup/guide.md:17]: timer

File: samples/ingestion/README.md (hunk 1 above)
Comment (HenryvanderVegte) [reply samples/ingestion/README.md:3]: ^Azure Speechis

File: samples/ingestion/ingestion-client/Setup/guide.md [hunk 5]
```diff
@@ -1,42 +1,42 @@
...
 
 The list of best practices we implemented as part of the solution are:
 
 1. Optimized the number of audio files included in each transcription with the view of achieving the shortest possible SAS TTL.
-2. Round Robin around selected regions in order to distribute load across available regions (per customer request)
-3. Retry logic optimization to handle smooth scaling up and transient HTTP 429 errors
-4. Running Azure Functions economically, ensuring minimal execution costs
+2. Retry logic optimization to handle smooth scaling up and transient HTTP 429 errors
+3. Running Azure Functions economically, ensuring minimal execution costs
+4. Round Robin around selected regions in order to distribute load across available regions (per customer request)
```
Comment (HenryvanderVegte) [inline samples/ingestion/ingestion-client/Setup/guide.md:24]: 4. is not supported by default!

File: samples/ingestion/ingestion-client/Setup/guide.md [hunk 6]
```diff
@@ -47,7 +47,7 @@ To test your the set up we recommend you use [Microsoft Azure Storage Explorer](
 
 ### Operating Mode
 
-Audio files can be processed either by the V3 API (batch) or our Speech SDK (real time). Both operations are wrapped and implemented in serverless and although seamless to the user there are sublte differences. In this section we enumerate those differences to serve as pros and cons of these options.
+Audio files can be processed either by the [V3 API (Batch)](https://centralus.dev.cognitive.microsoft.com/docs/services/speech-to-text-api-v3-0) or our [Speech SDK (real time)](https://docs.microsoft.com/en-us/azure/cognitive-services/speech-service/speech-sdk). Both operations are wrapped and implemented in serverless and although seamless to the user there are sublte differences. In this section we enumerate those differences to serve as pros and cons of these options.
```
Comment (HenryvanderVegte) [inline samples/ingestion/ingestion-client/Setup/guide.md:50]: subtle

Comment (HenryvanderVegte) [inline samples/ingestion/ingestion-client/Setup/guide.md:50]: the "Both operations are wrapped...." sentence sounds a bit weird imo

File: samples/ingestion/ingestion-client/Setup/guide.md [hunk 7]
```diff
@@ -64,9 +64,7 @@ In this mode audio files will be downloaded and streamed from the Azure Function
 
 1. Higher Azure Function costs [A single type Azure Functions will handle the process for each file and run at least for half the audio length]
 2. 2x processing of audio files [i.e. a 10-min file is transcribed in 5mins]
-3. There is no availability of the Diarization and Sentiment features in real time
-4. There is also no downstream SQL post processing option
-5. You will need to use the [Real Time ARM Template](ArmTemplateRealtime.json) from the repository for this operating mode.
+3. You will need to use the [Real Time ARM Template](ArmTemplateRealtime.json) from the repository for this operating mode.
```
Comment (HenryvanderVegte) [inline samples/ingestion/ingestion-client/Setup/guide.md:67]: Why did you remove 3. and 4.? they are not supported rn.

another bullet point would be that we only support WAV format for realtme right now, whereas batch supports other formats as well

File: samples/ingestion/ingestion-client/Setup/guide.md [hunk 8]
```diff
@@ -204,7 +200,7 @@ There are several containers to distinguish between the various outputs. We sugg
 
 ## Customizing the Ingestion Client
 
-By default, the ARM template uses the newest version of the Batch Ingestion Client which can be found in this repository. If a custom version should be used, the paths to the binaries inside the deployment template must be edited to point to a custom published version (by default, our binaries are: 
+By default, the ARM template uses the newest version of the Ingestion Client which can be found in this repository. If a custom version should be used, the paths to the binaries inside the deployment template must be edited to point to a custom published version (by default, our binaries are: 
 
 * https://mspublicstorage.blob.core.windows.net/transcription-enabled-storage/FetchTranscription.zip,
```
Comment (HenryvanderVegte) [inline samples/ingestion/ingestion-client/Setup/guide.md:205]: I was thinking that we maybe add the hash of the binary somewhere in there, in the docs and maybe even in the name of the file - so it's easier for users to check that the published code in this repo actually matches our precompiled functions

**Comment by alexeyo26** [comment]
@PanosPeriorellis A test comment to check

File: samples/ingestion/ingestion-client/Setup/guide.md (hunk 7 above)
Comment (PanosPeriorellis) [reply samples/ingestion/ingestion-client/Setup/guide.md:67]: i am writing only what is supported. I added the bullet on formats.

File: samples/ingestion/ingestion-client/Setup/guide.md (hunk 8 above)
Comment (PanosPeriorellis) [reply samples/ingestion/ingestion-client/Setup/guide.md:205]: ok?

Comment (PanosPeriorellis) [reply samples/ingestion/ingestion-client/Setup/guide.md:207]: later maybe?

**Review by HenryvanderVegte — APPROVED** [review]

**Review by HenryvanderVegte — APPROVED** [review]
